### PR TITLE
Shrshinde/isSupported-logs-check-v2

### PR DESCRIPTION
**Pull request by shrshindeMSFT**
- Adding isSupported check to logs capability
- Adding unit test case for isSupported

### CODE REVIEW

File: change/@microsoft-teams-js-d43a230c-d15c-486e-82c4-c72eb6ad2e61.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Adding isSupported Check to logs capability",
```
Comment (erinha) [inline change/@microsoft-teams-js-d43a230c-d15c-486e-82c4-c72eb6ad2e61.json:3]: nit: please change "Check" to lowercase "check" for consistency across change files

**Review by erinha — CHANGES_REQUESTED** [review]
Super small change, otherwise looks good! (Thanks for sending out a fresh PR, much easier to view the changes now!)

File: change/@microsoft-teams-js-d43a230c-d15c-486e-82c4-c72eb6ad2e61.json (hunk 1 above)
Comment (shrshindeMSFT) [reply change/@microsoft-teams-js-d43a230c-d15c-486e-82c4-c72eb6ad2e61.json]: Made changes in the latest commit.

**Review by erinha — APPROVED** [review]
🚀

**Review by MarcinJuraszek — APPROVED** [review]

**Pull request merged**